src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.lang.reflect.Array;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Application {
```

Comment:
The 'Application' class should not begin with an empty line according to the checklist.